test/e2e/suite/issuers/acme/certificate/http01.go

```diff
@@ -464,16 +463,20 @@ var _ = framework.CertManagerDescribe("ACME Certificate (HTTP01)", func() {
 		err = podClient.Delete(context.TODO(), pod.Name, metav1.DeleteOptions{})
 		Expect(err).NotTo(HaveOccurred())
 
+		By("Waiting for Certificate to exist")
+		cert, err = f.Helper().WaitForCertificateToExist(f.Namespace.Name, certificateSecretName, foreverTestTimeout)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
This looks wrong. Why wait for a *Certificate* with the name of the *Secret*? And why `foreverTestTimeout` ?

Reply:
Indeed, I fixed it

Reply:
BTW: forever means 60s apparently 🤣